phpunit.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be named phpunit.xml.dist. The phpunit.xml should be in .gitignore file.